contact.html

```diff
@@ -22,23 +24,24 @@
 		</div>
 		<div class="contactdetails">
 			<div class="contacticons">
-			<div class="mail"><a href="mailto:nickclifford@gmail.com">
-				<img alt="Email" src="mail.png" width="75" height="75">
+			<div id="mail"><a href="mailto:nickclifford@gmail.com">
+				<img alt="Email" src="mail.png" >
 				</a>
 			</div>
-			<div class="phone"><a href="tel:07970740492">
-				<img alt="Phone" src="phone.png" width="75" height="75">
+			<div id="phone"><a href="tel:07970740492">
+				<img alt="Phone" src="phone.png" >
 				</a>
 			</div>
-			<div class="twitter"><a href="https://twitter.com/cliffdeford">
-				<img alt="Twitter" src="twitter.png" width="75" height="75">
+			<div id="twitter"><a href="https://twitter.com/cliffdeford">
+				<img alt="Twitter" src="twitter.png">
 				</a>
 			</div>
 		</div>
 			<div class="contactform">
 			<h3>Or send me a message</h3>
 
 			<div class="formcontainer">
+				<form action="/nonexistent.php">
```

Comment:
Your indentation here is fine but the indentation of the rest of the form needs adjusting.